components/scream/CMakeLists.txt

```diff
@@ -43,6 +43,11 @@ set (CMAKE_CXX_STANDARD 11)
 if (NOT CIME_BUILD)
   project(SCREAM CXX Fortran)
 
+  if (SCREAM_CORI_HACK)
```

Comment:
Maybe we should simply probe the env, like Kokkos does. We believe (I think) that this issue is caused by Cray compiler wrapper, we could simply do `if ($ENV{CRAYPE_VERSION})`, avoiding misterious cache vars.

Reply:
That would be better except that we never established that it's a problem with Cray verses something specific to Cori (such as a bad  mixture of versions of cmake/intel etc). I don't think we have access to any other cray machine that works for scream so we can't prove our theory. In the absence of additional evidence, I'm reluctant to spread this very-hacky solution to other systems.

Reply:
Fair enough.